config/locales/en.yml

```diff
@@ -2904,7 +2905,6 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   first: "First"
   previous: "Previous"
   last: "Last"
-
```

Comment:
I dont think we should remove this line, this is the start of one of the major parts of this file, the spree key.